x-pack/plugins/lens/public/xy_visualization/state_helpers.ts

```diff
@@ -8,7 +8,11 @@ import { EuiIconType } from '@elastic/eui/src/components/icon/icon';
 import { SeriesType, visualizationTypes, LayerConfig, YConfig } from './types';
 
 export function isHorizontalSeries(seriesType: SeriesType) {
```

Comment:
I just noticed this for the first time, but I realized that you group horizontal series into it's own mixed chart type if there's multiple layers, but your using the Mixed XY chart icon. 

<img width="350" alt="Screen Shot 2020-09-08 at 16 18 40 PM" src="https://user-images.githubusercontent.com/549577/92525356-b0921f00-f1f1-11ea-8e87-809fb2b03616.png">


Can you change it to just display the Horizontal stacked bar chart icon?